Retro Cons on VCS chapter, consecutive headers (#8)
paulowoicho/the-turing-way: book/website/reproducible-research/vcs/vcs-git-branches.md

(rr-vcs-branches)=
# Git Branches

> When working on a project, individually or collaboratively, you may encounter the following issues:

Author: ```suggestion
When working on a project, individually or collaboratively, you may encounter the following scenarios:
```

- If you add a new feature to your project, you run the risk of accidentally breaking your working code as you test the feature. 
This would cause unexpected issues for active users of your project, even if the only active user is you.

> - When you collaborate with others, and everyone works on the master branch simultaneously, there will be a lot of confusion or conflicting changes.

Author: ```suggestion
- When you collaborate with others, and everyone works on the master branch simultaneously, there could be a lot of confusion and conflicting changes.
```

> - Some code/feature might not be interesting for everyone, and there needs to be a way to allow new work to be done on a project whilst protecting work that has already been done.

Author: ```suggestion
- Some code/feature might not be interesting for everyone. There may need to be a way to allow new work to be done on a project whilst protecting work that has already been done.
```
